perfkitbenchmarker/providers/aws/aws_virtual_machine.py

```diff
@@ -545,7 +545,10 @@ class RhelBasedAwsVirtualMachine(AwsVirtualMachine,
 
   def __init__(self, vm_spec):
     super(RhelBasedAwsVirtualMachine, self).__init__(vm_spec)
-    self.user_name = 'ec2-user'
+    if FLAGS.aws_user_name:
```

Comment:
Do you mean FLAGS['aws_user_name'].present?
Otherwise aws_user_name always have the default value ubuntu.